packages/devextreme/js/__internal/grids/grid_core/views/m_rows_view.ts

```diff
@@ -883,8 +886,8 @@ export class RowsView extends ColumnsView {
     return deferred;
   }
 
-  protected _getRows(change) {
-    return change && change.items || this._dataController.items();
+  protected _getRows(change?): Row[] {
```

Comment:
I think current function name is confusing, I am not sure if it's returning row elements or row item.
What do you think about renaming it to `_getRowItems`?